tools/rosbag/test/topic_publisher.py

```diff
@@ -0,0 +1,28 @@
+from std_msgs.msg import String
+from std_msgs.msg import Int32
+import rospy
+
+def topic_publisher():
```

Comment:
this should be part of the `test_custom_freq.py`.

Reply:
Then I will have to use multiprocess or multipthread since it will be while loop